packages/utils/src/requestdata.ts

```diff
@@ -250,6 +266,62 @@ export function extractRequestData(
   return requestData;
 }
 
+/**
+ * Add already normalized request data to an event.
+ */
+export function addNormalizedRequestDataToEvent(
+  event: Event,
+  req: Request,
+  // This is non-standard data that is not part of the regular HTTP request
+  additionalData: { ipAddress?: string; user?: Record<string, unknown> },
+  options: AddRequestDataToEventOptions,
+): Event {
```

Comment:
m: I would either make this function mutate the event OR return it, but not both. Having it do both makes it unclear from the outside whether it will mutate or return a new instance - especially if it is called `add...ToEvent`. Although I do realize that the other function does this too 🙄

Reply:
good point, we can do it better here. IMHO mutate only is probably the simplest!